ui/imports/shared/popups/BlockContactConfirmationDialog.qml

```diff
@@ -58,7 +54,7 @@ CommonContactDialog {
 
     StatusCheckBox {
         id: ctrlRemoveIDVerification
-        visible: contactDetails.isContact && !d.isTrusted && d.isVerificationRequestReceived
+        visible: (contactDetails.isContact && d.isTrusted) || contactDetails.trustStatus === Constants.trustStatus.trusted
```

Comment:
imo `contactDetails.isContact && d.isTrusted)` is sufficient

Reply:
Yeah I wanted to make double sure (also for Stovybook) :) It can't hurt and also I wasn't certain how this would get implemented in the end when I started working on this